hw/ip/kmac/dv/cov/kmac_masked_cov_excl.el

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Mon Jun 14 19:29:25 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "2274358225"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_rsp_gen
```

Comment:
Could this be an exclusion of the full tree under "tb.dut.u_tlul_adapter_msgfifo"? Also for "tb.dut.u_staterd.u_tlul_adapter"...

Reply:
This ties into a slightly more general topic I wanted to bring up in a meeting, let's circle back to this one afterwards.